Letterbook.Core/Letterbook.Core.csproj

```diff
@@ -8,6 +8,7 @@
 
     <ItemGroup>
       <PackageReference Include="Fedodo.ActivityPub.Model" Version="2.0.27" />
+      <PackageReference Include="Letterbook.ActivityPub" Version="1.0.0-ci0010-39" />
```

Comment:
Is there any reason we wouldn't want this to stay in sync with new [Letterbook.ActivityPub](https://github.com/Letterbook/Letterbook.ActivityPub) packages? Would it be worth investing in automating that to prevent drift between these projects?

Reply:
For now, yes they should stay in sync.